osdep/main-fn-win.c

```diff
@@ -80,8 +70,10 @@ int main(int argc_, char **argv_)
     if (argc > 0 && argv[0])
         MP_TARRAY_APPEND(NULL, argv_u8, argv_len, mp_to_utf8(argv_u8, argv[0]));
     if (gui) {
-        MP_TARRAY_APPEND(NULL, argv_u8, argv_len,
-                         "--player-operation-mode=pseudo-gui");
+        MP_TARRAY_APPEND(NULL, argv_u8, argv_len, "--player-operation-mode=pseudo-gui");
+        // Enable terminal output if attached to a console
+        if (cproc_count)
+            MP_TARRAY_APPEND(NULL, argv_u8, argv_len, "--terminal=yes");
```

Comment:
Wouldn't it break `--terminal=no` from user command line?

Reply:
No. User options are added last, so they're the ones that will ultimately be applied.